docs/docs/.vuepress/components/PythonMappings.vue

```diff
@@ -0,0 +1,193 @@
+<style lang="stylus"></style>
+
+<template>
+  <div class="md:w-3/4 m-auto">
+    <div
+      class="
+        bg-gray-100
+        py-8
+        px-4
+        w-full
+        border-l-8 border-green-500 border-solid
+      "
+    >
+      <h3 class="text-bold text-lg text-green-500 mb-2">
+        Don't see a package listed?
+      </h3>
+      Request to have a custom package mapping added:
+      <form
+        class="grid grid-cols-12"
+        v-on:submit.prevent="submitPackageRequest"
+      >
+        <input
+          v-model="requestedPackage"
+          placeholder="PyPi package name"
```

Comment:
Technically it is "PyPI" (capital i)

Reply:
(I didn't comment in other instances but I saw a few other "PyPi" references below)

Reply:
Ah interesting, **Py**thon **P**ackage **I**ndex.

In all the years I've used Python never made that connectio 😆